src/Middleware/Session/src/CookieProtection.cs

```diff
@@ -9,6 +10,8 @@ namespace Microsoft.AspNetCore.Session;
 
 internal static class CookieProtection
 {
+    [RequiresUnreferencedCode("This API uses Data Protection which may attempt to load types in a way that cannot be statically analyzed. " +
```

Comment:
@blowdart This is problematic and we don't have a trimmable approach for doing something like data protection as yet. Something to think about in the future.